### DISCUSSION TITLE
How can I show modal(or popup) when error occurred?

### DISCUSSION THREAD

**Discussion by hwangyena** [remix-run/react-router#10921] [Q&A]
I want to display an error popup while **keeping the page intact** when an error occurs. 
As far as I know, react-router provides only the errorElement component, which replaces the existing component with a new one. 

How can I achieve this?

**Comment by hwangyena** [comment]
I misunderstood React error boundaries, I will close this question.

**Reply by zaoan660** [reply]
I also want to implement this kind of feature. Do you know how to do it?
